pkgs/applications/editors/quilter/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv, fetchFromGitHub, fetchpatch, vala, pkgconfig, meson, ninja, granite
+, gtk3, desktop-file-utils, gnome3, gtksourceview, webkitgtk, gtkspell3, discount
+, gobjectIntrospection, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "quilter";
+  version = "1.6.3";
+
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "lainsce";
+    repo = pname;
+    rev = version;
+    sha256 = "1wa0i6dgg6fgb7q9z33v9qmn1a1dn3ik58v1f3a49dvd5xyf8q6q";
+  };
+
+  nativeBuildInputs = [
+    desktop-file-utils
+    gobjectIntrospection
+    meson
+    ninja
+    pkgconfig
+    vala
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    discount
+    granite
+    gtk3
+    gtksourceview
+    gtkspell3
+    webkitgtk
+    gnome3.libgee
+  ];
+
+  LIBRARY_PATH = stdenv.lib.makeLibraryPath [ discount ];
+
+  patches = [
+    # Fixes build with vala 0.42
+    (fetchpatch {
```

Comment:
While at it, could you upstream this as well?

Reply:
It has been https://github.com/lainsce/quilter/commit/a58838213cd7f2d33048c7b34b96dc887 and https://github.com/lainsce/quilter/commit/d1800ce830343a1715bc83da3339816554896be5.

(I'll switch to using these commits because at the time I thought it wasn't upstreamed)